tests/test_converter.py

```diff
@@ -22,14 +22,16 @@
 @pytest.mark.parametrize("package_type_column", [True, False])
 @pytest.mark.parametrize("explicit_column", [True, False])
 @pytest.mark.parametrize("merge_dependencies", MergeDependencies.__members__.values())
-@pytest.mark.parametrize("hide_tables", [True, False])
+@pytest.mark.parametrize("hide_tables", HideTables.__members__.values())
+@pytest.mark.parametrize("max_expanded_rows", [0, 10])
```

Comment:
let's actually stick to only testing against `[HideTables.no, HideTables.yes]` 😅 we already have way too many autogenerated files in this repo